### ISSUE TITLE
Latex content not being rendered when installed via npm

### ISSUE THREAD

**Issue by rorybyrne** [aaaakshat/gatsby-starter-texblog#2]
Hi, thanks for making this starter! I just tried to use it locally **using npm** and the LaTeX content didn't render at all.

Any ideas why?

![image](https://user-images.githubusercontent.com/9436784/103446469-f2027a80-4c77-11eb-93ff-f88576bbdfc4.png)

**Comment by bitmaks** [comment]
Hey, have you double checked your remark-math dependency with yarn? That has  caused me problems once in the past.

**Comment by rorybyrne** [comment]
What do you mean by that? Doesn't Gatsby install the dependency automatically? Do you recommend running a `npm i remark-math@x.y.z` command?

Also I'm using npm rather than yarn but that shouldnt be an issue right?

**Comment by rorybyrne** [comment]
I notice that deploying the website to Netlify via the one-click deployment seems to work fine - LaTeX renders correctly.

**Comment by bitmaks** [comment]
Yeah I'm not too sure, I haven't tried with npm. I cloned the repo right now with and built with yarn without any hiccups, I think it might be an npm compatibility issue possibly?

**Comment by rorybyrne** [comment]
It works via yarn. I'll update the issue title accordingly. Thank you!

**Comment by bitmaks** [comment]
I just tried the build with npm and I'm getting the same problem as you, something seems to be keeping the remark from working properly. I'll keep looking into this, let me know if you have any developments!

**Comment by miguelcarrasco** [comment]
Nice project! thanks for share it. I'm having the same issue with npm, maybe there are unmeet deps, I'll try to fix it.
